### DISCUSSION TITLE
Assign local B-H curves

### DISCUSSION THREAD

**Discussion by Maschinenausleger** [Eomys/pyleecan#575] [Q&A]
Hi everyone, 
I'm new to pyleecan, and was wondering if it's possible to assign different B-H curves to different parts of the rotor? Is there something like a spatial modifier? Or is it possible to separate the rotor into sections and assign other materials to every section?

Thank you

**Comment by SebGue** [comment]
Hello,

there is no spacial modifier.
Depending on what you actually want to do, you could use (customized) ventilation ducts. I.e. the ducts have their own material property 'mat_void' that can be any material you like.
If you on the other hand want to e.g. simulate laser or stamping edge effects with different edge layers, there need to be some code modifications.

Best regards, Sebastian
